u5-rest-apis/app.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const { json, urlencoded } = require('body-parser')
+
+const { readBooks, readBook, createBook, replaceBook, updateBook, deleteBook } = require('./feature/book-controller')
+const { readCovidR, readCovid, createCovid, updateCovid, deleteCovid } = require('./feature/covid-controller')
+
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+app.use(json())
+app.use(urlencoded({ extended: false }))
+
+// Routes
+app.get('/books', readBooks)
+app.get('/books/:isbn13', readBook)
+app.post('/books/', createBook)
+app.put('/books/:isbn13', replaceBook)
+app.patch('/books/:isbn13', updateBook)
+app.delete('/books/:isbn13', deleteBook)
+app.get('/covid', readCovidR)
+app.get('/covid/:stateId', readCovid)
```

Comment:
`R` หมายถึงอะไร?
ฉันคิดว่าคุณสามารถหาชื่อที่ดีกว่าสำหรับ `readCovidR`

What does the `R` mean?
I think you could find a better name for `readCovidR`.